packages/isomorphic/test/utils.test.ts

```diff
@@ -23,8 +23,12 @@ describe('utils', function () {
     expect(hexToBytes('DEADBEEF')).toEqual(new Uint8Array([222, 173, 190, 239]))
   })
 
-  it('bytesToHex - DEADBEEF', () => {
-    expect(bytesToHex([222, 173, 190, 239])).toEqual('DEADBEEF')
```

Comment:
Why was this one removed?

Reply:
@mvadari  reminder : did you mean the retain the existing test case and add more?

Reply:
Oops, typo

Reply:
Hm I'm not sure actually - either way I can add it back